backend-models/service-tests/accessedservice-tests.txt

```diff
@@ -0,0 +1,129 @@
+
+accessedservice.php
+
+-------------------
+
+********
+* pre-req
+********
+
+(checklogin() is TRUE && (hasAccess($userid, $cid, 'w') || isSuperUser($userid,) is TRUE))
+
+--------------------
+
+Test 1: try to update a created user by changing the username
+Row: 62 - 80 and 220 - 238
+Send {
+   $opt = "ADDUSER"
+   $uid = "UNK"
+   $regstatus = "UNK"
+}
+To make sure the update of a user works and what response it will get form the database.
+It can be done by creating a user with SQL-query.
+   SQL-query: insert into user( username, email, firstname, lastname, ssn, password, addedtime, class) 
+       VALUES ( :username, :email, :firstname, :lastname, :ssn, :password, now(), :class)
+
+Send {
+   $opt = "UPDATE"
+   $prop = "firstname"
+   $prop = "lastname"
+   $prop = "ssn"
+   $prop = "username"
+   $prop = "class"
+}
+Then make an update by useing SQL-query.
+   SQL-query: UPDATE user SET username=:username WHERE uid=:uid
+To update the username of the user. The values to be checked are username and uid.
+If the update was unable to succeed, it should display an error message which says 
+that the user does not exist in the database.
+Compare the result to make sure the test worked, by using SQL-query.
+   SQL-query: SELECT uid FROM user WHERE ssn=:ssn 
+
```

Comment:
you missed this entire section 

![image](https://user-images.githubusercontent.com/102600391/234272982-4166e683-5f42-4a41-8b67-e0344ae671b2.png)

the same as the privies comment apply.